gossip/src/epoch_slots.rs

```diff
@@ -45,8 +46,32 @@ impl Sanitize for Uncompressed {
 pub struct Flate2 {
     pub first_slot: Slot,
     pub num: usize,
-    #[serde(with = "serde_bytes")]
-    pub compressed: Vec<u8>,
+    #[serde(with = "serde_compat_bytes")]
+    pub compressed: Arc<Vec<u8>>,
```

Comment:
I guess there is no point for Arc here, the Flate2 version is not stored in agave AFAIK, it is only for transport.

Reply:
What do you mean?
You push the same CrdsValues to the cluster as the ones you store in your gossip CRDS table.

Reply:
Oh you mean if the CRDS gets forwarded to another node, we are not going to decompress it and will forward as-is? But would we then still need the CoW semantics? I guess it does not cost us anything here so we might just as well have it.

Reply:
I meant I do not understand this:
> the Flate2 version is not stored in agave AFAIK, it is only for transport.

values which are pushed over gossip are the same values that are stored in gossip table.